Documentation/Input/HowToAddNearInteractivity.md

```diff
@@ -12,7 +12,7 @@ Three key steps are required to listen for touch and/or grab input events on a p
 
 1. Ensure a [SpherePointer](Pointers.md#spherepointer) is registered in the *MRTK Pointer profile*.
 
-    The default MRTK profile and the default HoloLens 2 profile already contain a *SpherePointer*. One can confirm a SpherePointer will be created by selecting the MRTK Configuration Profile and navigating to **Input** > **Pointers** > **Pointer Options**. The default [GrabPointer](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Pointers/GrabPointer.prefab) prefab, under *MRTK.SDK/Features/UX/Prefabs/Pointers/*, should be listed with a *Controller Type* of *Articulated Hand*. A custom prefab can be utilized as long as it implements the [`SpherePointer`](xref:Microsoft.MixedReality.Toolkit.Input.SpherePointer) class.
+    The default MRTK profile and the default HoloLens 2 profile already contain a *SpherePointer*. One can confirm a SpherePointer will be created by selecting the MRTK Configuration Profile and navigating to **Input** > **Pointers** > **Pointer Options**. The default `GrabPointer` (Assets/MRTK/SDK/Features/UX/Prefabs/Pointers) prefab should be listed with a *Controller Type* of *Articulated Hand*. A custom prefab can be utilized as long as it implements the [`SpherePointer`](xref:Microsoft.MixedReality.Toolkit.Input.SpherePointer) class.
```

Comment:
nit: 
default `GrabPointer` (Assets/MRTK/SDK/Features/UX/Prefabs/Pointers) prefab 
default `GrabPointer` prefab (Assets/MRTK/SDK/Features/UX/Prefabs/Pointers)

Reply:
the general pattern has been 'prefab' after the reference, but i will move.